src/nspawn/nspawn.c

```diff
@@ -990,7 +990,7 @@ static int mount_cgroup_hierarchy(const char *dest, const char *controller, cons
         to = strjoina(dest, "/sys/fs/cgroup/", hierarchy);
 
         r = path_is_mount_point(to, false);
-        if (r < 0)
+        if (r < 0 && r != -ENOENT)
```

Comment:
This condition looks like this in the upstream code as well. For some reason, `path_is_mount_point()` can check paths that do not exist.